app.js

```diff
@@ -233,6 +233,31 @@ $("menu_top_right_contents").addEventListener("click", event => {
 });
 
 //
+// UI events to set the font size
+//
+$("decrease_font_size").addEventListener("click", event => {
+  var font_size_input = $("input_fontsize");
+  if (font_size_input.value > 0) {
+    font_size_input.value = parseInt(font_size_input.value)-1;
```

Comment:
Nit: whitespace around `-`.
Also, perhaps reset the size to some default value if `parseInt` fails.